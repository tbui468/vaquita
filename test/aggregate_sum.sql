if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        ("Venus", 2.32, 0, 92.0, false),
        ("Earth", null, 1, 1.0, true),
        ("Mars", 4.23, 2, 0.01, false),
        ("Saturn", 4.23, 2, 0.01, false);

select ring_system, sum(number_of_moons) from planets group by ring_system;
select ring_system, sum(10) from planets group by ring_system;
close sol;
drop database sol;
exit;
