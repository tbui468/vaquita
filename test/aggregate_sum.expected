created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
false, 4
true, 1
false, 30
true, 10
closed database sol
dropped database sol
